constants/Profile/SupProfileHeader.js

```diff
@@ -0,0 +1,121 @@
+import { withTheme } from "react-native-elements";
+
+export default {
+  container: {
+    height: 270,
+    flexDirection: "column"
+  },
+  rightContainer: {
+    flexDirection: "column"
+  },
+  leftContainer: {
+    flex: 0,
+    width: "30%",
+    height: "40%",
+    alignItems: "center"
+  },
+  textContainer: {
+    paddingTop: 2,
+    height: 61
+  },
+  titleText: {
+    marginTop: 22,
+    lineHeight: 22,
+    fontSize: 20,
+    fontFamily: "Lato-Bold",
+    color: "white"
+  },
+  userText: {
+    lineHeight: 19,
+    fontSize: 16,
+    fontFamily: "Lato",
+    color: "white"
+  },
+  // socialContainer  : {

```

Comment:
Same here, we can remove this

Reply:
This file also deprecated and should be removed from branch unless some changes made need to be noted